test/test_coretasks.py

```diff
@@ -17,6 +17,7 @@
 [core]
 owner = Uowner
 nick = TestBot
+auth_password = hunter2
```

Comment:
Why is that?

Reply:
Missed when moving tests to the other file